platform/avr-rcb/contiki-rcb-main.c

```diff
@@ -109,21 +109,22 @@ init_lowlevel(void)
   rs232_redirect_stdout(RS232_PORT_1);
   
   DDRE |= LED1 | LED2 | LED3;
+  
+  ctimer_init();
+ 
 }
 
 
 static struct etimer et;
 PROCESS_THREAD(rcb_leds, ev, data)
 {
-  uint8_t error;
 
   PROCESS_BEGIN();
   
-  if((error = icmp6_new(NULL)) == 0) {
     while(1) {
       PROCESS_YIELD();
       
-#if NETSTACK_CONF_WITH_IPV6	  
+#if UIP_CONF_IPV6	  
```

Comment:
You are right, that´s nonsense. I accidentally changed that macro because of a copy and paste mistake.